kubernetes/schema_pod_spec.go

```diff
@@ -356,12 +356,12 @@ func podSpecFields(isUpdatable, isComputed bool) map[string]*schema.Schema {
 								"gmsa_credential_spec": {
 									Type:        schema.TypeString,
 									Description: "GMSACredentialSpec is where the GMSA admission webhook inlines the contents of the GMSA credential spec named by the GMSACredentialSpecName field",
-									Optional:    true,
+									Required:    true,
```

Comment:
This field and `GMSACredentialSpecName` are [optional](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1WindowsSecurityContextOptions.md) . 

However, it looks like currently, omitting this settings will still be enforced to be empty string instead of being omitted from the API or set to `null`. Should I report a separate issue?